analyzers/src/SonarAnalyzer.CSharp/Rules/DeclareTypesInNamespaces.cs

```diff
@@ -47,18 +47,15 @@ public sealed class DeclareTypesInNamespaces : DeclareTypesInNamespacesBase<Synt
             ((BaseTypeDeclarationSyntax)declaration).Identifier;
 
         protected override bool IsInnerTypeOrWithinNamespace(SyntaxNode declaration, SemanticModel semanticModel)
-        {
-            switch (declaration.Parent.Kind())
+            => declaration.Parent.Kind() switch
```

Comment:
Coding style: The `=>` token should be moved to the previous line for the sake of consistency.

Reply:
Done.